tests/e2e/unified_test.go

```diff
@@ -122,12 +126,60 @@ func TestAllInSeries(t *testing.T) {
 
 				if os.Getenv("E2E_GCP_TARGET") == "vcr" {
 					// Stop recording after tests finish and write to cassette
+					vcrRecorder := h.Rec
 					defer func() {
-						err := h.Rec.Stop()
+						err := vcrRecorder.Stop()
 						if err != nil {
 							t.Errorf("[VCR] Failed stop vcr recorder: %v", err)
 						}
 					}()
+
+					hook := func(i *cassette.Interaction) error {
```

Comment:
Cool - this looks like what I was looking for in my comments on the other PR.  Can you reverse these also?  The other option is to "fix" {uniqueID} when running in VCR mode, I.e. always have the unique ID be 11111111111 or something like that